lib/features/manage_account/presentation/vacation/vacation_controller.dart

```diff
@@ -92,6 +94,16 @@ class VacationController extends BaseController {
     });
   }
 
+  void _initFocusListener() {
+    subjectTextFocusNode.addListener(() {
+      if (subjectTextFocusNode.hasFocus == true) {
+        if (PlatformInfo.isMobile) {
+          richTextControllerForMobile.hideRichTextView();
+        }
+      }
+    });
```

Comment:
Create method to call `subjectTextFocusNode.removeListener` later in `onClose()`. Avoid using anonymous function inside `listener`.

Reply:
done